Postman Test Suite/Ed-Fi ODS-API Profile Test Suite.postman_collection.json

```diff
@@ -5457,6 +5457,69 @@
 				{
 					"name": "Setup",
 					"item": [
+						{
+							"name": "Extension Descriptors",
+							"item": [
+								{
+									"name": "Ensure Art Medium Descriptor Exists",
```

Comment:
This creates the descriptor values used by the test if they aren't already present (perhaps due to the failure to execute the ODS migration script for the sample extension).